.github/workflows/validate-mdx.links.yaml

```diff
@@ -0,0 +1,31 @@
+name: validate-mdx-links
```

Comment:
This could be moved to `shared` if we see a future where we reuse this in other repositories.

Reply:
I'm in the process of moving it now.